tests/console/validate_packages_and_patterns.pm

```diff
@@ -39,6 +39,7 @@ $software{'update-test-feature'} = {    # See poo#36451
     repo => is_sle('15+') ? 'Basesystem' : 'SLES',
     installed => 0,
     available => sub { get_var('BETA') },
+    condition => sub { is_sle('<15-SP6') },
```

Comment:
One minor comment, can you please add the reason why we skip the test on sle15sp6?
```
# skip the check on sle15sp6 due to bsc#1225970
```

Reply:
Thanks, comment added